core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -18,11 +18,18 @@
 
 // Service for handling all interactions with the exploration editor backend.
 oppia.factory('explorationData', [
-  '$http', '$log', 'alertsService',
+  '$http', '$log',
```

Comment:
Hm, in a way, having one dependency per line is neat, but it's weird for this to be the only place in the entire codebase where this happens. Could you bring it back to how it was, where we try to use most of the line width?

Also, in the process, it'd be great if you could move $q to near the beginning as well, so that all the $-dependencies come first in the list. Thanks!